.ahoy.yml

```diff
@@ -78,14 +78,9 @@ commands:
 
   lint:
     usage: Lint code
-    cmd: docker-compose exec -T test ./vendor/bin/govcms-lint web/themes/custom
-
-  tests:
-    usage: Run all tests.
     cmd: |
-      ahoy test-behat
-      ahoy test-phpunit
-      ahoy lint
+      docker-compose exec -T test ./vendor/bin/govcms-lint web/modules/custom
```

Comment:
due to some limitations in `./vendor/bin/govcms-lint` script, only a single argument can be passed.

We are calling this twice - once for modules and once for themes. It is better than what we currently have - just calling it for themes.